docs/source/index.rst

```diff
@@ -41,16 +42,13 @@ DAML SDK documentation
    :caption: Building applications
 
    app-dev/app-arch
-   app-dev/authentication
-   app-dev/ledger-api
+   json-api/index
+   daml2js/index
    app-dev/bindings-ts/index
-   app-dev/bindings-java/index
```

Comment:
What is happening to those? You seem to have removed them from the index but it looks like you didn’t believe the files.

Reply:
Ah I missed that we have a separate index for the ledger API, thanks for the explanation!